by-language/scala-spark-http/SparkCrateDBhttpExample.scala

```diff
@@ -0,0 +1,64 @@
+import scalaj.http.{Http, HttpOptions}
+import org.apache.spark.sql.{SparkSession, Row}
+import org.apache.spark.sql.types._
+import org.json4s.jackson.Serialization
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+import java.sql.Timestamp
+
+object SparkCrateDBhttpExample {
+  def main(args: Array[String]): Unit = {
+    val formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm")
+	
+	val data = Seq(
+	  Row(Timestamp.valueOf(LocalDateTime.parse("2024-07-10 10:00", formatter)), Array(9.744417, 47.413417), """{"example_quantity_field": 30, "example_string_field": "abc"}"""),
+	  Row(Timestamp.valueOf(LocalDateTime.parse("2024-07-10 11:00", formatter)), Array(13.46738, 52.50463), """{"example_quantity_field": 40, "example_string_field": "def"}""")
+	)
+	
+	val spark = SparkSession.builder
+		.appName("test")
+		.master("local[*]") 
+		.getOrCreate()
+	val df = spark.createDataFrame(
+	  spark.sparkContext.parallelize(data),
```

Comment:
Sweet. Does `spark.sparkContext.parallelize()` offer any options for configuration, or is it "auto-parcelling" only?